apps/web/pages/getting-started.tsx

```diff
@@ -150,7 +150,7 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
   const bioRef = useRef<HTMLInputElement>(null);
   /** End Name */
   /** TimeZone */
-  const [selectedTimeZone, setSelectedTimeZone] = useState(user.timeZone ?? dayjs.tz.guess());
+  const [selectedTimeZone, setSelectedTimeZone] = useState(dayjs.tz.guess());
```

Comment:
The actual problem is this https://github.com/calcom/cal.com/blob/main/packages/prisma/schema.prisma#L138
There is a default timezone set for all users. So, `user.timeZone` is always set even if it is not set by user.

`user.timeZone` check is actually required because user might abandon onboarding in between after selecting his timezone. This allows him to not redo this step of setting the timezone.

Because we need to first fix the default in schema.prisma, it looks like the right fix to me till then(plus for users who have their account created already and are on onboarding step). Because it is a bigger problem to have wrong timezone shown by default to new users.

cc @zomars I don't think there should be a problem in removing the default timezone in schema.prisma.